src/test/groovy/org/prebid/server/functional/tests/AmpSpec.groovy

```diff
@@ -180,4 +183,79 @@ class AmpSpec extends BaseSpec {
     }
+
+    def "PBS should pass addtl_consent to user.ext.{consented_providers_settings/ConsentedProvidersSettings}.consented_providers"() {
+        given: "Default amp request with addtlConsent"
+        def randomAddtlConsent = PBSUtils.randomString
+        def ampRequest = AmpRequest.defaultAmpRequest.tap {
+            addtlConsent = randomAddtlConsent
+        }
+
+        and: "Save storedRequest into DB"
+        def ampStoredRequest = BidRequest.defaultBidRequest.tap {
+            user = new User(ext: new UserExt(
+                    consentedProvidersSettingsKebabCase: new ConsentedProvidersSettings(consentedProviders: PBSUtils.randomString),
+                    consentedProvidersSettings: new ConsentedProvidersSettings(consentedProviders: PBSUtils.randomString)))
+        }
+        def storedRequest = StoredRequest.getStoredRequest(ampRequest, ampStoredRequest)
+        storedRequestDao.save(storedRequest)
+
+        when: "PBS processes amp request"
+        defaultPbsService.sendAmpRequest(ampRequest)
+
+        then: "Bidder request should contain addtl consent"
+        def bidderRequest = bidder.getBidderRequest(ampStoredRequest.id)
+        assert bidderRequest.user.ext.consentedProvidersSettingsKebabCase.consentedProviders == randomAddtlConsent
+        assert bidderRequest.user.ext.consentedProvidersSettings.consentedProviders == randomAddtlConsent
+    }
+
+    def "PBS should leave requested user.ext.{consented_providers_settings/ConsentedProvidersSettings}.consented_providers as is"() {
+        given: "Default amp request with addtlConsent"
+        def ampRequest = AmpRequest.defaultAmpRequest.tap {
+            addtlConsent = null
+        }
+
+        and: "Save storedRequest into DB"
+        def consentProvidersKebabCase = PBSUtils.randomString
+        def consentProviders = PBSUtils.randomString
+        def ampStoredRequest = BidRequest.defaultBidRequest.tap {
+            user = new User(ext: new UserExt(
+                    consentedProvidersSettingsKebabCase: new ConsentedProvidersSettings(consentedProviders: consentProvidersKebabCase),
+                    consentedProvidersSettings: new ConsentedProvidersSettings(consentedProviders: consentProviders)))
+        }
+        def storedRequest = StoredRequest.getStoredRequest(ampRequest, ampStoredRequest)
+        storedRequestDao.save(storedRequest)
+
+        when: "PBS processes amp request"
+        defaultPbsService.sendAmpRequest(ampRequest)
+
+        then: "Bidder request should contain requested consent"
+        def bidderRequest = bidder.getBidderRequest(ampStoredRequest.id)
+        assert bidderRequest.user.ext.consentedProvidersSettingsKebabCase.consentedProviders == consentProvidersKebabCase
+        assert bidderRequest.user.ext.consentedProvidersSettings.consentedProviders == consentProviders
+    }
+
+    def "PBS shouldn't pass addtl_consent to user.ext.{consented_providers_settings/ConsentedProvidersSettings}.consented_providers"() {
```

Comment:
PBS should left user.ext.{consented_providers_settings/ConsentedProvidersSettings}.consented_providers empty when  addtl_consent and original fields are empty